developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md

```diff
@@ -0,0 +1,199 @@
+
+## Initialize local configuration and create an account
+
+Follow the steps below to [start Aptos and create an account](../../../guides/get-test-funds.md) on the blockchain. Once done, you will have a local `.aptos/` directory containing a `config.yaml` configuration file that is used to store configurations between CLI runs. This is local to your run, so you will need to continue running the Aptos CLI from this directory or reinitialize in another directory.
+
+### Step 1: Run Aptos init
+
+The `aptos init` command initializes the configuration with the private key you provide or generates anew if not given:
+
+```bash
+$ aptos init
+Configuring for profile default
+Enter your rest endpoint [Current: None | No input: https://fullnode.devnet.aptoslabs.com]
+
+No rest url given, using https://fullnode.devnet.aptoslabs.com...
+Enter your faucet endpoint [Current: None | No input: https://faucet.devnet.aptoslabs.com]
+
+No faucet url given, using https://faucet.devnet.aptoslabs.com...
+Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+
+No key given, generating key...
+Account 00f1f20ddd0b0dd2291b6e42c97274668c479bca70f07c6b6a80b99720779696 does not exist, creating it and funding it with 10000 coins
+Aptos is now set up for account 00f1f20ddd0b0dd2291b6e42c97274668c479bca70f07c6b6a80b99720779696!  Run `aptos help` for more information about commands
+
+{
+  "Result": "Success"
+}
+  ```
+### Stept 2: Create other profiles
+
+You can also create other profiles for different endpoints and different keys. Do this by including the `--profile` argument and the value of your choosing to the `aptos init` command as shown below. Note that argument can be used in most other commands to replace command line arguments.
+
+```bash
+$ aptos init --profile superuser
+Configuring for profile superuser
+Enter your rest endpoint [Current: None | No input: https://fullnode.devnet.aptoslabs.com]
+
+No rest url given, using https://fullnode.devnet.aptoslabs.com...
+Enter your faucet endpoint [Current: None | No input: https://faucet.devnet.aptoslabs.com]
+
+No faucet url given, using https://faucet.devnet.aptoslabs.com...
+Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+
+No key given, generating key...
+Account 18B61497FD290B02BB0751F44381CADA1657C2B3AA6194A00D9BC9A85FAD3B04 does not exist, creating it and funding it with 10000 coins
+Aptos is now set up for account 18B61497FD290B02BB0751F44381CADA1657C2B3AA6194A00D9BC9A85FAD3B04!  Run `aptos help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+
+### Stept 3: Connect to Aptos network
+
+To create a delegation pool and obtain information about it, connect to the [Aptos Network](./connect-to-aptos-network.md) and launch your own Aptos node.
+
+## Initialize a delegation pool
```

Comment:
You can actually use that to get the address prior to the delegation pool being created

Reply:
It seems to me that you can't get the right address prior that way. Because the contract uses additional module salt in the creation - https://github.com/aptos-labs/aptos-core/blob/442e608e689eb842fee427ee3af257e1a4d70205/aptos-move/framework/aptos-framework/sources/delegation_pool.move#L101 and https://github.com/aptos-labs/aptos-core/blob/442e608e689eb842fee427ee3af257e1a4d70205/aptos-move/framework/aptos-framework/sources/delegation_pool.move#L421-L425. Or am I wrong?

Reply:
Sure, let's add it @michelle-aptos. I removed the bit about it needing to be done after the delegation pool has been created (based on Greg's comments). @gregnazario can you please verify Krishnacore's observation? The PR is ready, but I'd like to verify this bit before we publish it as is. Thanks!

Reply:
Thank you, @gregnazario and @Krishnacore! I updated the section to take both scenarios into consideration.